app.js

```diff
@@ -105,11 +105,11 @@ function validateArgs(url, opts, cb) {
       cb(new error.BackupError('InvalidOption', 'Invalid URL host.'));
       return;
     }
-    if (!urlObject.pathname) {
+    if (!urlObject.pathname || urlObject.pathname === '/') {
       cb(new error.BackupError('InvalidOption', 'Invalid URL, missing path element (no database).'));
       return;
     }
-    if (opts && opts.iamApiKey && urlObject.auth) {
```

Comment:
Is this `auth` referenced anywhere in the docs that should also be removed?

Reply:
No, the `urlObject` is completely internal, but I searched anyway and can see there is a related `auth` issue from the earlier legacy URL changes in the example scripts. I guess I'll tidy that up here too.

Reply:
Done in 44227bf